src/Role/Services/DefaultRoleEditingService.php

```diff
@@ -47,24 +46,14 @@ public function create(StringLiteral $name): UUID
         $uuid = new UUID($this->uuidGenerator->generate());
         $command = new CreateRole(
             $uuid,
-            $name
+            $name->toNative()
         );
 
         $this->commandBus->dispatch($command);
 
         return $uuid;
     }
 
-    public function rename(UUID $uuid, StringLiteral $name): void
```

Comment:
I think more methods can be removed? Or will you remove the complete file in the final pull request?

Reply:
They have to stay there because the current controller still needs them. They will indeed be removed in the final PR when the switch is made to use the new handlers iso the controller. This function specifically was never used, even in the current controller which dispatch directly on the commandbus

Reply:
True! 😅